source/solvers/heat_transfer_scratch_data.cc

```diff
@@ -209,6 +212,23 @@ HeatTransferScratchData<dim>::calculate_physical_properties()
                 this->phase_values[q],
                 this->grad_specific_heat_temperature_0[q],
                 this->grad_specific_heat_temperature_1[q]);
+
+              // Coefficient used to neglect viscous dissipation in the fluid
+              // which is more than 10 times less viscous than the other fluid
+              if (this->viscosity_1[q] > 10. * this->viscosity_0[q])
```

Comment:
Why do you want to neglect the viscous dissipation automatically in the fluid which is less viscous? Especially because here viscosity is the kinematic viscosity. For example, if I have a liquid with rho=1000 and mu =1 and air with rho =1 and mu = 0.01, I know that my air will have negligible viscous dissipation when compared to my liquid (liquid will be 100x larger). Yet this comparison will give you air as the viscous dissipater. I would do the comparison on  the dynamic viscosity and not the viscosity. 

In this case, I  think it would be more relevant to just specify from a user parameter on which phase viscous dissipation is applied (if you want to enable for both). The threshold for 10 seems a bit arbitrary. Might as well specify the fluid indicator with a default value of 1 and let it be this way :).

Reply:
Yes the user-defined option was also on my mind, I did not want to add yet another parameter, but I agree that it will be really less hacky than hard-coding a threshold. So I will change that!